web-console/knikubevirt/Create-vm/create-vm.md

```diff
@@ -0,0 +1,189 @@
+
+If the wizard cannot connect to the vCenter instance using the credentials stored within its Secret, an error message will be displayed. The user will need to fix the Secret’s credentials before continuing.
+
+
+![vCenter new instance](img/Step-1-basic-import-6.png)
+
+The user can also connect to a new vCenter instance by clicking Connect to new instance from within the dropdown.
+
+![Checking credentials](img/Step-1-basic-import-7.png)
+
+New fields for vCenter URL, vCenter Username, vCenter Password, and VM to import appear along with a checkbox to “Remember vCenter credentials” that is unchecked by default.
+
+The vCenter URL field should include a syntax hint with the desired formatting. The user is likely to paste a variety of URLs, so any FQDN the user submits should be validated and automatically corrected before being used to communicate with vCenter’s API.
+
+When the user clicks the “Check” button a text string appears below the button with a spinner to indicate that the credentials are being checked.
+
+If the wizard fails to connect to vCenter using the provided credentials, an error similar to the one above for an invalid vCenter instance should be displayed directly below the vCenter Password field.
+
+
+![vcenter connected](img/Step-1-basic-import-8.png)
+
+If the connection to vCenter succeeds, the first (alphabetical) VM along with its OS, Flavor, and Workload Profile are automatically selected and filled in as usual. The “Remember vCenter credentials” checkbox also becomes enabled to allow the user to save them as a new Kubernetes Secret.
+
+With the target virtual machine selected, the user can either quickly skip to Step 5 “Review” by clicking the “Import Virtual Machine” secondary action button, or click “Next” to proceed through the rest of the wizard.
+
+
+### ISO
+
+![ISO](img/Step-1-basic-ISO.png)
+
+ISO enables the user to install an Operating System from existing installation media within ContainerDisk.
+
+![ISO selected](img/Step-1-basic-ISO-2.png)
+
+The user must select an ISO and then manually specify the OS, Flavor, and Workload Profile. None of those fields can be automatically pre-filled.
+
+### Image
+
+![Image](img/Step-1-basic-Image-1.png)
+
+Image enables the user to deploy an appliance or OS from an existing OVF file from ContainerDisk.
+
+![Image selected](img/Step-1-basic-Image-2.png)
+
+With an OVF selected, the OS is automatically filled and locked. The user will still need to choose a Flavor and Workload Profile, and can customize the image via cloud-init.
+
+### Appliance
+
+![appliance](img/Step-1-basic-appliance-1.png)
+
+Appliance enables the user to deploy an appliance or OS from an existing Appliance within ContainerDisk.
+
+![appliance selected](img/Step-1-basic-appliance-2.png)
+
+With an appliance selected, the OS is automatically filled in and locked. The user will still need to choose a Flavor and Workload Profile.
+
+### PXE
+
+![PXE](img/Step-1-basic-PXE-1.png)
```

Comment:
Maybe tweak the hint text to call out that "Networking" is an upcoming step.

> NIC "nic 1" will be used. This can be changed in the Networking step.